sql/src/main/java/io/crate/operation/user/User.java

```diff
@@ -57,14 +58,28 @@ public String name() {
         return privileges;
     }
 
+    public boolean isSuperUser() {
+        return roles.contains(Role.SUPERUSER);
+    }
+
+    public boolean hasPrivilege(Privilege.Type type, Privilege.Clazz clazz, Privilege.State state) {
+        Optional<Privilege> hasPrivilege = privileges.stream()
```

Comment:
@seut but technically i don't have all the information such as grantor etc .. but i can always use `crate` for grantor and null for the ident